src/mono/mono/utils/monobitset.h

```diff
@@ -50,7 +50,7 @@ enum {
     MonoBitSet *tmp_src = (src); \
     MonoBitSet *tmp_dest = (dest); \
     int i, size; \
```

Comment:
```suggestion
    size_t i, size; \
```

Then the cast can be removed.